Readme.md

```diff
@@ -1,6 +1,9 @@
-# Biotope
+# Biotope, a toolchain for lasting design systems.
 ![shield build](https://img.shields.io/travis/biotope/biotope.svg?label=build)
 ![shield npm](https://img.shields.io/npm/v/biotope.svg)
+Biotope helps you to build, distribute and adopt a library of web components with a consistent design and of high quality across all your websites and apps. With Biotope you can easily collaborate with your team to develop and maintain the user interface of all your digital experiences from a single location.
```

Comment:
```suggestion

Biotope helps you to build, distribute and adopt a library of web components with a consistent design and of high quality across all your websites and apps. With Biotope you can easily collaborate with your team to develop and maintain the user interface of all your digital experiences from a single location.
```
Could you add a line break here to start the text on a new line after the badges?